src/Modules/Workspaces/Commands.Workspaces/help/Get-PowerBIWorkspace.md

```diff
@@ -111,7 +120,7 @@ Aliases: Top
 
 Required: False
 Position: Named
-Default value: 5000
+Default value: 100
```

Comment:
Does this persist if you do another build again or does it change the default value back?

Reply:
It does persist upon multiple builds